lib/min_heap.rb

```diff
@@ -8,26 +8,29 @@ def initialize(key, value)
 end
 
 class MinHeap
-
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(logn)
+  # Space Complexity: O(n)
```

Comment:
Since the additional space `add` causes is due to the 1 item being added to the heap and the log n recursive calls (call stack), this method's space complexity is O(log n)